test/conftest.py

```diff
@@ -327,7 +327,13 @@ def fixture_myhoard_config(default_backup_site, mysql_master, session_tmpdir):
         "http_port": get_random_port(start=3000, end=30000),
         "mysql": {
             "binlog_prefix": mysql_master.config_options.binlog_file_prefix,
-            "client_params": {"host": "127.0.0.1", "password": "NgLqvU8gbWCtfJWJPy", "port": 3306, "user": "root"},
+            "client_params": {
+                "host": "127.0.0.1",
+                "password": "NgLqvU8gbWCtfJWJPy",
```

Comment:
It was already in the source, but I'd add a comment to precise that it's fake.
If it's not it must be changed + rotated.

Reply:
It's fake (same one that is used in `myhoard.json` file).